samples/nrf9160/modem_shell/README.rst

```diff
@@ -259,6 +289,10 @@ You can use the socket tool to:
 * Create and manage socket connections.
 * Send and receive data.
 
+.. note::
+   When both normal (CP-)RAI and 3GPP Release 14 AS-RAI are enabled, which can be the case for NB-IOT,
+   then both are signalled. It depends on the network configuration and prioritization, which RAI takes affect.
```

Comment:
Should we just say "CP-RAI" instead of "normal (CP-)RAI"? Or is "normal RAI" clear for everybody? :)
```suggestion
   When both CP-RAI and 3GPP Release 14 AS-RAI are enabled, which can be the case for NB-IoT,
   then both are signalled. It depends on the network configuration and prioritization, which RAI takes affect.
```

Reply:
Discussed with Jouni and decided to open up the terminology a bit more:
`When both 3GPP Release 13 Control Plane (CP) RAI and 3GPP Release 14 Access Stratum (AS) RAI are enabled.`
Please resolve if this is fine.

Reply:
Yes, the terminology-wise it's clear now. Does signalled mean here "requested from the network"? Also this paragraph seems a bit out of place here IMO. Could it be placed in the end, after RAI example?

Reply:
I don't remember exactly the signaling anymore as it's been quite a while but there is at least RAI support entry which network sends while LTE connection is negotiated with the network.

I haven't been able to find a good place for this text. We haven't had much text within the examples but there are some. I've moved it now and need to wait how it looks after document build is completed.

Reply:
I guess this is now fine in the generated document. Please close if fine.